src/go/rpk/pkg/cli/cmd/cloud/login.go

```diff
@@ -24,13 +25,38 @@ func NewLoginCommand(fs afero.Fs) *cobra.Command {
 		Short: "Login to vectorized cloud",
 		Long:  `Authorize rpk to access vectorized cloud with your user credentials.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			// try to look for existing token in config first
+			configRW := config.NewVCloudConfigReaderWriter(fs)
+			token, err := configRW.ReadToken()
+			if err != nil && err != config.ErrConfigFileDoesNotExist {
+				return fmt.Errorf("error reading token from config file: %w", err)
+			}
+
 			auth0Client := vcloud.NewDefaultAuth0Client()
-			token, err := vcloud.Login(auth0Client)
+			if err == nil {
```

Comment:
You could add a clause for the case when `err == config.ErrConfigFileDoesNotExist` before this, so you don't have to wrap all this in an if (following Go's convention of returning early).

```go
			if err != nil && err != config.ErrConfigFileDoesNotExist {
				return fmt.Errorf("error reading token from config file: %w", err)
			}
			if err == config.ErrConfigFileDoesNotExist {
				// handle stuff
			}
			// rest of the code
```